deploy/helm/sumologic/upgrade-2.0.0.sh

```diff
@@ -15,6 +15,7 @@ readonly MAX_YQ_VERSION=4.0.0
 readonly KEY_MAPPINGS="
 prometheus-operator.prometheusOperator.tlsProxy.enabled:kube-prometheus-stack.prometheusOperator.tls.enabled
 otelcol.deployment.image.name:otelcol.deployment.image.repository
+fluent-bit.image.fluent_bit:fluent-bit.image.repository
```

Comment:
Is this correct? I thought `fluent-bit.image.fluent_bit` maps to `fluent-bit.image` and not to `fluent-bit.image.repository`?

Reply:
Good call, it's wrong. Let me fix that.

Reply:
Please update tests according to change :sweat_smile:

Reply:
Done.